docs/framework/unmanaged-api/profiling/icorprofilerinfo8-isfunctiondynamic-method.md

```diff
@@ -43,7 +43,7 @@ A function is considered dynamic if it has no metadata. Certain methods like IL
 
 **Library:** CorGuids.lib
 
-**.NET Framework Versions:** [!INCLUDE[net_current_v472plus](../../../../includes/net-current-v472plus.md)
+**.NET Framework Versions:** [!INCLUDE[net_current_v472plus](../../../../includes/net-current-v472plus.md)]  
```

Comment:
I don't think we need the extra spaces here

```suggestion
**.NET Framework Versions:** [!INCLUDE[net_current_v472plus](../../../../includes/net-current-v472plus.md)]
```

Reply:
Thanks @mairaw , both changes were asked by the bugfixing team but we can try the fix without the hard breakline